### PR TITLE
Replace settings error slug with polylang

### PR DESCRIPTION
**Pull request by manooweb**
Fixes partially https://github.com/polylang/polylang-pro/issues/2033 Polylang part.

### CODE REVIEW

File: settings/settings.php
Comment (Hug0-Drelon) [inline settings/settings.php:1]: Should we take the opportunity to use `esc_html__()` instead of `__()` in this file?

Comment (manooweb) [reply settings/settings.php:1]: It's another task of #2023

**Review by Chouby — COMMENTED** [review]
Shouldn't this test fail ? https://github.com/polylang/polylang/blob/3.5.4/tests/phpunit/tests/test-settings.php#L109-L119

**Comment by manooweb** [comment]
> Shouldn't this test fail ? https://github.com/polylang/polylang/blob/3.5.4/tests/phpunit/tests/test-settings.php#L109-L119

No! Because notices in this case are displayed by WordPress without using any notice slug.
So we can set any notice slug we want here.

See https://github.com/WordPress/WordPress/blob/6.4.3/wp-admin/options-head.php#L18 used [here](https://github.com/polylang/polylang/blob/3.5.4/settings/view-languages.php#L12)

**Pull request merged**